doc/source/release-notes/Percona-Server-5.5.52-38.3.rst

```diff
@@ -0,0 +1,53 @@
+.. rn:: 5.5.52-38.3
+
+============================
+|Percona Server| 5.5.52-38.3
+============================
+
+Percona is glad to announce the release of |Percona Server| 5.5.52-38.3 on
+October 4th, 2016. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.5/Percona-Server-5.5.52-38.3/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+
+Based on `MySQL 5.5.52
+<http://dev.mysql.com/doc/relnotes/mysql/5.5/en/news-5-5-52.html>`_, including
+all the bug fixes in it, |Percona Server| 5.5.52-38.3 is now the current stable
+release in the 5.5 series. All of |Percona|'s software is open-source and free,
+all the details of the release can be found in the `5.5.52-38.3 milestone at
+Launchpad <https://launchpad.net/percona-server/+milestone/5.5.52-38.3>`_.
+
+Bugs Fixed
+==========
+
+ ``mysql_upgrade`` would fail when server was running with
+ :variable:`gtid_mode` set to ``ON`` and ``--disable-gtid-unsafe-statements``
+ because system tables are stored using ``MyISAM``. Bug fixed :bug:`14221043`
+ (upstream :mysqlbug:`56155`).
+
+ :ref:`audit_log_plugin` would hang when trying to write a log record of
+ :variable:`audit_log_buffer_size` length. Bug fixed :bug:`1588439`.
+
+ After fixing bug :bug:`1540338`, system table engine validation check is no
+ longer skipped for tables that don't have an explicit ``ENGINE`` clause in
+ a ``CREATE TABLE`` statement. If |MySQL| upgrade statements are replicated,
+ and slave does not have the |MyISAM| set as a default storage engine, then
+ the ``CREATE TABLE mysql.server`` statement would attempt to create an
+ |InnoDB| table and fail. Bug fixed :bug:`1600056`.
```

Comment:
and fail, because mysql_system_tables.sql script omitted explicit engine setting for this table.